ess/src/API/EMBC.ESS/Resources/Evacuations/Contract.cs

```diff
@@ -170,6 +166,16 @@ public enum NeedsAssessmentType
         Assessed
     }
 
+    public enum IdentifiedNeed
+    {
+        ShelterReferral,
```

Comment:
I think these are supposed to be two options on one field, not two fields

Reply:
Actually, we can't make it a single field - it is 'grouped' in the front end, but eventually it is one or the other that will be enforced in the UI.